cmd/crank/beta/trace/trace.go

```diff
@@ -126,6 +127,11 @@ func (c *Cmd) Run(k *kong.Context, logger logging.Logger) error {
 	}
 	logger.Debug("Found kubeconfig")
 
+	// XXX: this needs to be made configurable - see TODO on line 64
+	// I used the values below for checking timing as concurrency increases
+	// kubeconfig.QPS = 50
```

Comment:
this commented out section will probably be replaced, at least for the time being until it's configurable, by https://github.com/crossplane/crossplane/pull/5737